node/narwhal/ledger-service/src/ledger.rs

```diff
@@ -134,6 +134,20 @@ impl<N: Network, C: ConsensusStorage<N>> LedgerService<N> for CoreLedgerService<
         }
     }
 
+    /// Returns the previous committee for the given round.
+    /// If the previous round is in the future, then the current committee is returned.
+    fn get_previous_committee_for_round(&self, round: u64) -> Result<Committee<N>> {
+        // Get the round number for the previous committee. Note, we subtract 2 from even rounds,
+        // because committees are updated in odd rounds.
+        let previous_round = match round % 2 == 0 {
```

Comment:
Technically, `let previous_round = round.saturating_sub(2);` should suffice as well. Since an `odd_round - 2` and `odd_round - 1` would reference the same committee.